apps/website/src/server/db/short-links.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from "zod";
+import { TRPCError } from "@trpc/server";
+import { prisma } from "@/server/db/client";
+import { convertToSlug, SLUG_REGEX } from "@/utils/slugs";
+
+export const shortLinkSchema = z.object({
+  label: z.string(),
+  slug: z.string().regex(SLUG_REGEX).optional(),
+  link: z.string(),
```

Comment:
I think we should require a valid link here. Probably zod's `z.string().url()` should work fine.